### PR TITLE
Locales: Fix placeholder inconsistencies

### PR DESCRIPTION
**Pull request by fnetX**
Address #25238

This fixes all placeholder inconsistencies in translations that I have found, some of them probably don't change anything visible, but they're inconsistent nevertheless.

I do not know how RTL text and placeholders work together, so these strings are left to check:

~~~
options/locale/locale_tr-TR.ini check how rtl and strings correlate
options/locale/locale_fa-IR.ini:delete_tag=برچسب %[2] از <a href="%[1]s">%[3]</a> حذف شدند
options/locale/locale_fa-IR.ini:delete_branch=شاخه %[2] از <a href="%[1]s">%[3]</a> حذف شدند
options/locale/locale_fa-IR.ini:issues.ref_reopening_from=`<a href="%[3]s"> تقاضای واکشی ارجاع شده %[4] که مسائله بازگشایی خواهد کرد</a> <a id="%[1]s" href="#%[1]s">%[2] </a>`
options/locale/locale_fa-IR.ini:issues.ref_from=`از %[1]`
options/locale/locale_fa-IR.ini:monitor.queue.settings.maxnumberworkers.placeholder=در حال حاضر %[1]v
options/locale/locale_fa-IR.ini:issue_assigned.pull=@%[1]s به شما برای درخواست pull %[2]s در ریپازیتوری %[3] محول شده.
options/locale/locale_fa-IR.ini:delete_tag=برچسب %[2] از <a href="%[1]s">%[3]</a> حذف شدند
options/locale/locale_fa-IR.ini:delete_branch=شاخه %[2] از <a href="%[1]s">%[3]</a> حذف شدند
options/locale/locale_fa-IR.ini:issues.ref_issue_from=`<a href="%[3]s"> ارجاعات این مسائله %[4]</a> <a id="%[1]s" href="#%[1]s">%[2]s</a>`
options/locale/locale_fa-IR.ini:issues.ref_pull_from=`<a href="%[3]s"> ارجاعات این تقاضای ادغام %[4]</a> <a id="%[1]s" href="#%[1]s">%[2]s</a>`
options/locale/locale_fa-IR.ini:issues.ref_closing_from=`<a href="%[3]s"> ارجاعات این تقاضای واکشی %[4]</a> <a id="%[1]s" href="#%[1]s">%[2]s</a>`
options/locale/locale_fa-IR.ini:issues.ref_reopening_from=`<a href="%[3]s"> تقاضای واکشی ارجاع شده %[4] که مسائله بازگشایی خواهد کرد</a> <a id="%[1]s" href="#%[1]s">%[2] </a>`
options/locale/locale_fa-IR.ini:issues.ref_closed_from=`<a href="%[3]s"> بسته شده این مسائله %[4]</a> <a id="%[1]s" href="#%[1]s">%[2]s</a>`
options/locale/locale_fa-IR.ini:issues.ref_reopened_from=`<a href="%[3]s"> بازگشایی این مسائله %[4]</a> <a id="%[1]s" href="#%[1]s">%[2]s</a>`
~~~

Also, in options/locale/locale_pt-BR.ini:
~~~
issues.due_date_remove=removeu a data limite %s %s
~~~

I do not know if this works as intended. I suppose it does, but other languages do it differently.

I still recommend that a proper QA for translation placeholders is implemented, especially checking that all placeholders are there. AFAIK, Weblate does this, it is Free Software. I have no experience with the proprietary Crowdin platform.

Also check out the corresponding @Forgejo issue at https://codeberg.org/forgejo/forgejo/issues/863 - maybe there are some helpful discussions, too. I'll probably rather interact there.

### CODE REVIEW

**Comment by lunny** [comment]
Thank you for your contribution, but we cannot merge this changes into translation files otherwise they may conflict with daily update from crowdin. But we can change them in crowdin follow your PR.

**Comment by wxiaoguang** [comment]
At the moment, the fixes could be done on Crowdin side.

After the INI package refactoring, I think there could be some tools to help to figure out/fix translation errors.

**Comment by lunny** [comment]
Fixed in crowdin:

cs-CZ
- https://crowdin.com/translate/gitea/68/enus-cs?filter=basic&value=0#q=activate_account.text_1
- https://crowdin.com/translate/gitea/68/enus-cs?filter=basic&value=0#q=issue_assigned.pull
- https://crowdin.com/translate/gitea/68/enus-cs?filter=basic&value=0#q=issue_assigned.issue
- https://crowdin.com/translate/gitea/68/enus-cs?filter=basic&value=0#q=members.remove.detail

de-DE
- https://crowdin.com/translate/gitea/68/enus-de?filter=basic&value=0#q=monitor.queue.settings.maxnumberworkers.placeholder

el-GR
- https://crowdin.com/translate/gitea/68/enus-el?filter=basic&value=0#q=activate_account.text_1
- https://crowdin.com/translate/gitea/68/enus-el?filter=basic&value=0#q=team_invite.text_1
- https://crowdin.com/translate/gitea/68/enus-el?filter=basic&value=0#q=dashboard.task.unknown

fa-IR
- https://crowdin.com/translate/gitea/68/enus-fa?filter=basic&value=0#q=issue_assigned.pull

is-IS
- https://crowdin.com/translate/gitea/68/enus-is?filter=basic&value=0#q=issues.opened_by_fake

lv-LV
- https://crowdin.com/translate/gitea/68/enus-lv?filter=basic&value=0#q=team_invite.text_1

nl-NL
- https://crowdin.com/translate/gitea/68/enus-nl?filter=basic&value=0#q=issue_assigned.issue
- https://crowdin.com/translate/gitea/68/enus-nl?filter=basic&value=0#q=issue.action.merge

pl-PL
- https://crowdin.com/translate/gitea/68/enus-pl?filter=basic&value=0#q=monitor.queue.settings.maxnumberworkers.placeholder

pt-BR
- https://crowdin.com/translate/gitea/68/enus-ptbr?filter=basic&value=0#q=issue_assigned.pull
- https://crowdin.com/translate/gitea/68/enus-ptbr?filter=basic&value=0#q=issues.due_date_modified
- https://crowdin.com/translate/gitea/68/enus-ptbr?filter=basic&value=0#q=dashboard.task.error

ru-RU
- https://crowdin.com/translate/gitea/68/enus-ru?filter=basic&value=0#q=team_invite.subject

si-LK
- https://crowdin.com/translate/gitea/68/enus-silk?filter=basic&value=0#q=issue.action.merge
- https://crowdin.com/translate/gitea/68/enus-silk?filter=basic&value=0#q=issue.action.new
- https://crowdin.com/translate/gitea/68/enus-silk?filter=basic&value=0#q=release.new.text
- https://crowdin.com/translate/gitea/68/enus-silk?filter=basic&value=0#q=migrated_from
- https://crowdin.com/translate/gitea/68/enus-silk?filter=basic&value=0#q=issues.ref_issue_from
- https://crowdin.com/translate/gitea/68/enus-silk?filter=basic&value=0#q=issues.force_push_codes
- https://crowdin.com/translate/gitea/68/enus-silk?filter=basic&value=0#q=members.remove.detail
- https://crowdin.com/translate/gitea/68/enus-silk?filter=basic&value=0#q=dashboard.task.cancelled
- https://crowdin.com/translate/gitea/68/enus-silk?filter=basic&value=0#q=push_tag
- https://crowdin.com/translate/gitea/68/enus-silk?filter=basic&value=0#q=delete_tag
- https://crowdin.com/translate/gitea/68/enus-silk?filter=basic&value=0#q=delete_branch
- https://crowdin.com/translate/gitea/68/enus-silk?filter=basic&value=0#q=approve_pull_request

sk-SK
- https://crowdin.com/translate/gitea/68/enus-sk?filter=basic&value=0#q=issue_assigned.pull
- https://crowdin.com/translate/gitea/68/enus-sk?filter=basic&value=0#q=issue.action.new

tr-TR
- https://crowdin.com/translate/gitea/68/enus-tr?filter=basic&value=0#q=team_invite.subject
- https://crowdin.com/translate/gitea/68/enus-tr?filter=basic&value=0#q=team_invite.text_1
- https://crowdin.com/translate/gitea/68/enus-tr?filter=basic&value=0#q=team_invite.text_3

uk-UA
- https://crowdin.com/translate/gitea/68/enus-uk?filter=basic&value=0#q=issues.ref_issue_from
- https://crowdin.com/translate/gitea/68/enus-uk?filter=basic&value=0#q=issues.ref_closing_from
- https://crowdin.com/translate/gitea/68/enus-uk?filter=basic&value=0#q=issues.ref_reopening_from
- https://crowdin.com/translate/gitea/68/enus-uk?filter=basic&value=0#q=issues.force_push_codes
- https://crowdin.com/translate/gitea/68/enus-uk?filter=basic&value=0#q=push_tag
- https://crowdin.com/translate/gitea/68/enus-uk?filter=basic&value=0#q=approve_pull_request
- https://crowdin.com/translate/gitea/68/enus-uk?filter=basic&value=0#q=review_dismissed
- https://crowdin.com/translate/gitea/68/enus-uk?filter=basic&value=0#q=watched_repo

And one key should be removed from all translation files.
- dashboard.cron.cancelled

**Comment by wxiaoguang** [comment]
Made another change, because it's RTL.

https://crowdin.com/translate/gitea/68/enus-fa?filter=basic&value=0#q=issue_assigned.pull

image

**Comment by 6543** [comment]
I'll merge main into this untill diff = 0 :)

**Comment by wxiaoguang** [comment]
The crowdin fixes are in main now

**Comment by 6543** [comment]
5 things are missing ...

**Comment by wxiaoguang** [comment]
Only one is missing `dashboard.task.unknown`

Others are right (not wrong) on main branch.

File: options/locale/locale_si-LK.ini [hunk 1]
```diff
@@ -2117,7 +2117,7 @@ dashboard.task.finished=කාර්යය:%[1]s[2]s විසින් ආර
 dashboard.task.unknown=නොදන්නා කාර්යය:%[1]s
 dashboard.cron.started=ආරම්භ Con:%[1]s
 dashboard.cron.process=ක්රෝන්:%[1]s
-dashboard.cron.cancelled=Con: %s අවලංගු:%[3]ගේ
```
Comment (6543) [inline options/locale/locale_si-LK.ini:2120]: @wxiaoguang is'nt that missing a type (s)

Comment (wxiaoguang) [reply options/locale/locale_si-LK.ini:2120]: Yes, this key in en-US has changed, so it triggers re-translation, the `dashboard.cron.cancelled` won't exist in main branch IMO

**Comment by 6543** [comment]
https://crowdin.com/translate/gitea/68/enus-el?filter=basic&value=0#q=dashboard.task.unknown

**Comment by wxiaoguang** [comment]
I have made some changes:

* el-GR: dashboard.task.unknown : https://crowdin.com/translate/gitea/68/enus-el?filter=basic&value=0#q=dashboard.task.unknown
* si-LK: dashboard.cron.cancelled : https://crowdin.com/translate/gitea/68/enus-silk?filter=basic&value=0#q=dashboard.cron.cancelled
    * I think this change will not take affect because the newly synchronized key from en-US will discard the out-dated translations soon. But it's fine to make a change to close this issue.

**Pull request closed without merging**